MetaMask/metamask-extension
# Update for a new Add network page

Fixes: #13679

Explanation:  
UX update for a new "Add network" screen in storybook.

Manual testing steps:  
  -  Start storybook
  -  Find Add network component in the storybook
  - Check a new design for the `Add network` screen
  
  Figma: 
  https://www.figma.com/file/wfOwLmE52OKY0bSrsQwHPL/Add-popular-custom-network?node-id=1697%3A17291
  
  
<img width="545" alt="image" src="https://user-images.githubusercontent.com/97883527/157186263-5c9ae251-de32-4040-801f-4c0aa4fe1b12.png">
